BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -84,19 +88,20 @@ extension ElementContent {
     }
 
     /// Initializes a new `ElementContent` with no children that delegates to the provided measure function.
-    public init(measureFunction: @escaping (SizeConstraint) -> CGSize) {
+    public static func withMeasurement(_ measureFunction: @escaping (SizeConstraint) -> CGSize) -> ElementContent {
```

Comment:
Swapped these out b/c the two init methods that take closures cause conflicts which confuse the compiler. Pushing them behind a static func makes the intention clearer and prevents future issues.

Reply:
These can be disambiguated by:
- providing the argument types, e.g. `ElementContent { constraint -> CGSize in ... }`
- not using a trailing closure

I'm not as keen on the static funcs; they're not very discoverable with all the inits on `ElementContent`. It's kind of weird to have `ElementContent(measurable: Measurable)` as an init but for the closure version to be a static factory.